### PR TITLE
chore(cardinal): expose AddToQueue in Cardinal 

### PR DESCRIPTION
**Pull request by namrapatel**
Closes: WORLD-431

## Brief Changelog

- [x] Expose `ecs/transaction.go`'s `AddToQueue` in `cardinal.World`

## Testing and Verifying

We are ignoring the fact that we don't have great tests for `cardinal.World` for the time being.

### CODE REVIEW

**Review by pyrofolium — APPROVED** [review]
lgtm

**Comment by pyrofolium** [comment]
I think people will be opinionated on this one might stir up opinions. So wait on it before merge.

File: cardinal/transaction.go
Comment (jerargus) [inline cardinal/transaction.go:48]: Can't you just make this t.impl.AddToQueue and leave out the change to cardinal/ecs/world.go?

**Review by jerargus — APPROVED** [review]
I don't think should be added to the top level cardinal package as a user shouldn't be using this method directly. Also, generating signatures is probably going to be confusing in normal usage. But since we don't have an alternate plan to allow for testing transactions, I think it's fine.

Please add a comment that this method is only for testing.

File: cardinal/transaction.go
Comment (namrapatel) [reply cardinal/transaction.go]: Yeah, great idea, done!

**Review by jerargus — APPROVED** [review]

**Pull request merged**
